app/controllers/users_controller.rb

```diff
@@ -67,8 +67,11 @@ def handle_ip
     # Prevent someone from gaming the site by referring themselves.
     # Presumably, users are doing this from the same device so block
     # their ip after their ip appears three times in the database.
+    # Using split.first in case client is behind proxy to get his ip.
+    # If app is deployed on standalone server use request.remote_ip to
+    # get client ip address.
 
-    address = request.env['HTTP_X_FORWARDED_FOR']
+    address = request.env['HTTP_X_FORWARDED_FOR'].try(:split).try(:first) || request.remote_ip
```

Comment:
OK, good to know. 

I think if we use `fetch` we can safely drop the two `try`. Slightly more readable to me, happy to accept a working alternative though.

    address = request.env.fetch('HTTP_X_FORWARDED_FOR', request.remote_ip).split.first.gsub(',', '')